fix builtins armhf default arch detection by adding the same armhf triple
handling use for compiler-rt
Index: llvm-toolchain-snapshot_14~++20220105111900+cdbad62c526c/compiler-rt/lib/builtins/CMakeLists.txt
===================================================================
--- llvm-toolchain-snapshot_14~++20220105111900+cdbad62c526c.orig/compiler-rt/lib/builtins/CMakeLists.txt
+++ llvm-toolchain-snapshot_14~++20220105111900+cdbad62c526c/compiler-rt/lib/builtins/CMakeLists.txt
@@ -27,6 +27,11 @@ if (CMAKE_SOURCE_DIR STREQUAL CMAKE_CURR
 
   load_llvm_config()
   construct_compiler_rt_default_triple()
+  if ("${COMPILER_RT_DEFAULT_TARGET_TRIPLE}" MATCHES ".*hf$")
+    if (${COMPILER_RT_DEFAULT_TARGET_ARCH} MATCHES "^arm")
+      set(COMPILER_RT_DEFAULT_TARGET_ARCH "armhf")
+    endif()
+  endif()
 
   include(SetPlatformToolchainTools)
   if(APPLE)
